pkg/machines/components/create-vm-dialog/autoDetectOS.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/python3
+
+import gi
+gi.require_version('Libosinfo', '1.0')
+from gi.repository import Libosinfo
```

Comment:
This requires adding a dependency (Recommends) to python(3)-gobject-base, plus the corresponding package that actually ships the Typelib; this needs to happen in tools/cockpit.spec and tools/debian/control.

Does that typelib exist on RHEL 7? Supposedly on these OSes, this should just fail and autodetection will simply be unsupported (which is fine).

Reply:
Yep, it's part of RHEL-7. I've updated the deps according to your review.